packages/puppeteer-core/src/common/bidi/Page.ts

```diff
@@ -549,6 +550,224 @@ export class Page extends PageBase {
     }
   }
 
+  #exposedFunctionId = Math.random();
+  override async exposeFunction<Args extends unknown[], Ret>(
+    name: string,
```

Comment:
That's okay, because one gets overwritten by the other and the other never gets used after. What we should do is detect when these get cleaned up, but there is no reasonable way at the moment.

Reply:
Write a `TODO` as this is not expected behavior compare to the CDP implementation.